org-formation/organization.yaml

```diff
@@ -72,23 +75,35 @@ Organization:
       Accounts:
         - !Ref ITSandboxAccount
 
-  SynapseOU:
+  SynapseProdOU:
```

Comment:
changing SynapseOU and BridgeOU requires a corresponding change to SCP policy references, https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/cfc4327ae461a5877076e97714d3297fd5b07c54/org-formation/010-scps/_tasks.yaml#L19

we might just be able to remove them from the SCP file since they are under the `PlatformOU` which means that they will inherit SCP from the `PlatformOU`

Reply:
Good catch... I agree that removing `SynapseOU` and `BridgeOU` should be removed and the SCPs applied via `PlatformOU` is best

Reply:
@ahayden do you plan on making the changes to the SCPs to match the changes in this organizations file?